sys/posix/include/semaphore.h

```diff
@@ -34,6 +34,9 @@
 extern "C" {
 #endif
 
+/**
+ * @brief A sema_t is a sem_t is a sema_t
```

Comment:
looking through the code it seems that RIOT has `sema_t` but POSIX hat `sem_t`, and there is also `sys_sem_t`.